content/docs/practices/index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: Best practices
-sort_rank: 8
+sort_rank: 9
```

Comment:
I think this needs to stay the same, as this is about the sort order of the "Best practices" menu item among the top-level items.